### ISSUE TITLE
Erro after force macOS shutdown

### ISSUE THREAD

**Issue by brunos0** [dail8859/NotepadNext#650]
### Operating System and Version

Macos 14.7

### Distribution

Dmg file

### Description

I'm using 0.8 version on my macos 14.7. 
Yesterday, after my job o force my macos tô shutdown and when i triedntonuse notepad next ungot the erro from image.

Sorry, but i can sendo more information because my macos was a security policies that Prevent me to copy more data that i send here.

By the way there some way to copy temporário (nota saved file), to backup my data?
Maybe remove the app and ALL data could bem fiz my problem.

Kind regards.

### Steps to Reproduce

1. Install notepad 0.8 on macos 14.7
2. Create some filés on It
3. While app is openned, try to shutdown the macos without close notepad next

### Additional Details

e.g. screenshots, logs, etc
![IMG_20241018_093321681](https://github.com/user-attachments/assets/1da86775-c8cd-490f-8a64-d2608c59876a)

**Comment by brunos0** [comment]
I found a solution.

On macos ALL unsaved files was temporarily keeps at
/Users/Xxxxxx/Library/Application Support/Notepad next/session/

I copied ALL filés tô another folder and deleted ALL files on that source folder.
Then i was able to open the Application without error.

Now Just need tô open ALL backup filés and get previous data.

I Hope that this helps someone tô Prevent loose yours filés after a force shutdown.

**Comment by bsdooby** [comment]
Great you found a solution for your issue. Maybe mine is related to yours (?) ; see #649

**Comment by dail8859** [comment]
> On macos ALL unsaved files was temporarily keeps at
> /Users/Xxxxxx/Library/Application Support/Notepad next/session/

Yes the temporary files are saved to disk. I cannot say for sure why it was giving the error but there was most likely something corrupted in the `ini` file that saves all of Notepad Next's settings (including information about those session files). If you do have this happen again I would be _very_ interested in looking into the ini file more to make sure there is plenty of error catching to prevent this kind of error.

**Comment by matthewyang204** [comment]
Then we could simply have an exception to this error so that when the `ini` file is corrupted, we could simply load with default settings? Or should we simply autosave the ini file to a backup, so that if it's crashed and the `ini` file is corrupted, copy from backup?

**Comment by dail8859** [comment]
@matthewyang204 Yeah alot more robustness and ultimately some sort of general exception handling would be a good idea as there's been more than once case of users completely unable to load the application to an apparent config file issue.

**Comment by alsyundawy** [comment]
try

https://github.com/alsyundawy/NotepadNext-MacOS
